integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1017,6 +1017,15 @@ func (s *DockerSuite) TestRunApparmorProcDirectory(c *check.C) {
 	}
 }
 
+// test that non existent apparmor profile gives helpful error
+func (s *DockerSuite) TestRunApparmorProfileNotFound(c *check.C) {
+	testRequires(c, SameHostDaemon, Apparmor)
+
+	out, _, err := dockerCmdWithError("run", "--security-opt", "apparmor=thisapparmorprofiledoesnotexistonthismachine", "busybox", "ls")
+	c.Assert(err, checker.NotNil, check.Commentf(out))
+	c.Assert(out, checker.Contains, "Error response from daemon: Apparmor profile not found.")
```

Comment:
Should there be a full-stop at the end here? (perhaps just check for "Apparmor profile not found")